integration-tests/README.md

```diff
@@ -31,18 +31,50 @@ docker info
 Running Integration tests
 =========================
 
-## Running tests using mvn
+## Starting docker tests
 
-To run all the tests using mvn run the following command -
+To run all the tests using docker and mvn run the following command -
 '''''
   mvn verify -P integration-tests
 '''''
 
-To run only a single test using mvn run following command -
+To run only a single test using mvn run the following command -
 '''''
   mvn verify -P integration-tests -Dit.test=<test_name>
 '''''
 
+## Configure and run integration tests using existing cluster
```

Comment:
the ITTwitterQueryTest and ITWikipediaQueryTest needs some segment entries to be made in mysql, which is done during the docker setup. Anyone using a custom cluster will also need to make those entries manually in metadata storage. (docker/sample-data.sql has those entries.) This needs to be added to the docs.